public/docs/_examples/template-syntax/dart/lib/app_component.dart

```diff
@@ -19,7 +19,17 @@ enum Color { Red, Green, Blue }
       MyClickDirective,
       MyClickDirective2
     ])
-class AppComponent {
+class AppComponent implements OnInit, AfterViewInit {
+  @override
+  ngOnInit() {
```

Comment:
The lifecycle methods don't have return types (here and below) is this intentional?